tests/camera.cpp

```diff
@@ -0,0 +1,57 @@
+
+/* Copyright (c) 2015, Human Brain Project
+ *                     Daniel Nachbaur <danielnachbaur@epfl.ch>
+ */
+
+#define BOOST_TEST_MODULE camera
+
+#include <boost/test/unit_test.hpp>
+
+#include <zerobuf/render/camera.h>
+
+
+BOOST_AUTO_TEST_CASE(emptyCamera)
+{
+    const zerobuf::render::Camera camera;
+    const std::vector< float > empty( 3, 0 );
+    const std::vector< float >& origin = camera.getOriginVector();
+    const std::vector< float >& lookAt = camera.getLookAtVector();
+    const std::vector< float >& up = camera.getUpVector();
```

Comment:
In a later commit we should add default values so the camera can have a meaningful default setup.